projects/packages/publicize/legacy/enhanced-open-graph.php

```diff
@@ -115,30 +140,37 @@ function enhanced_og_video( $tags ) {
 
 	if ( preg_match( '/((youtube|vimeo)\.com|youtu.be)/', $video_url ) ) {
 		if ( strstr( $video_url, 'youtube' ) ) {
-			$id = jetpack_get_youtube_id( $video_url );
-			$video_url = 'http://www.youtube.com/embed/' . $id;
+			$id               = jetpack_get_youtube_id( $video_url );
+			$video_url        = 'http://www.youtube.com/embed/' . $id;
 			$secure_video_url = 'https://www.youtube.com/embed/' . $id;
-		} else if ( strstr( $video_url, 'vimeo' ) ) {
+		} elseif ( strstr( $video_url, 'vimeo' ) ) {
 			preg_match( '|vimeo\.com/(\d+)/?$|i', $video_url, $match );
-			$id = (int) $match[1];
-			$video_url = 'http://vimeo.com/moogaloop.swf?clip_id=' . $id;
+			$id               = (int) $match[1];
+			$video_url        = 'http://vimeo.com/moogaloop.swf?clip_id=' . $id;
 			$secure_video_url = 'https://vimeo.com/moogaloop.swf?clip_id=' . $id;
 		}
 	}
 
 	$tags['og:video']            = $video_url;
 	$tags['og:video:secure_url'] = $secure_video_url;
 
-	if ( empty( $post->post_title ) )
+	if ( empty( $post->post_title ) ) {
+		/* translators: %s is the name of the site */
 		$tags['og:title'] = sprintf( __( 'Video on %s', 'jetpack' ), get_option( 'blogname' ) );
+	}
 
 	return $tags;
 }
 add_filter( 'jetpack_open_graph_tags', 'enhanced_og_video' );
 
+/**
+ * Check if a post has a suitable featured image.
+ *
+ * @param int $post_id The post ID to check.
+ * @return bool True if the post has a suitable featured image, false otherwise.
+ */
 function enhanced_og_has_featured_image( $post_id ) {
 	$featured = Jetpack_PostImages::from_thumbnail( $post_id, 200, 200 );
-	if ( !empty( $featured ) && count( $featured ) > 0 )
-		return true;
-	return false;
+
+	return ! empty( $featured ) && count( $featured ) > 0;
```

Comment:
Is `$featured` an array? If so then this could just be 
```suggestion
	return ! empty( $featured );
```

Perhaps even removing the need to assign it to `$featured`, so we could do:

```php
    return ! empty( Jetpack_PostImages::from_thumbnail( $post_id, 200, 200 ) );
```